#  zero divison error
# print(2/0)

# try: 
#     print(2/0)
# except:
#     print("Sorry, can't give off that output")

try:
    print(5/0)
except:
    pass
else:
    print("A value was passed...")

